api/scripts/database/create-database.js

```diff
@@ -20,6 +20,8 @@ PgClient.getClient(url.href).then(async (client) => {
   } catch (error) {
     if (error.code === PGSQL_DUPLICATE_DATABASE_ERROR) {
       logger.info(`Database ${DB_TO_CREATE_NAME} already created`);
+    } else {
+      logger.error(`Could not be done because ${error.detail}`);
```

Comment:
```suggestion
      logger.error(`Database creation could not be done because ${error.detail}`);
```
ou
```suggestion
      logger.error(`Database creation failed: ${error.detail}`);
```